app/src/main/java/com/vultisig/wallet/ui/screens/ChainCoinScreen.kt

```diff
@@ -228,34 +240,55 @@ fun ChainCoinScreen(navController: NavHostController) {
 private fun CoinListHeader(
     address: String,
     name: String,
-    totalBalance: String
+    totalBalance: String,
+    explorerURL: String,
 ) {
-    val appColor = MaterialTheme.appColor
+    val context = LocalContext.current
+    val appColor = Theme.colors
     val dimens = MaterialTheme.dimens
+    val uriHandler = LocalUriHandler.current
     LaunchedEffect(key1 = Unit) {
         Timber.d(address)
     }
     Column(modifier = Modifier.padding(vertical = 20.dp, horizontal = 16.dp)) {
         Row(verticalAlignment = Alignment.CenterVertically) {
             Text(
                 text = name,
-                style = Theme.montserrat.headlineMedium,
+                style = Theme.montserrat.titleLarge,
                 color = appColor.neutral0
             )
             Spacer(modifier = Modifier.width(16.dp))
-            Icon(
-                painter = painterResource(id = R.drawable.copy),
-                contentDescription = null
-            )
+            IconButton(onClick = {
+                val clipboard =
+                    context.getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager
+                val clip = ClipData.newPlainText("address", address)
+                clipboard.setPrimaryClip(clip)
```

Comment:
Compose LocalClipboardManager can be used

```suggestion
                val clipboard = LocalClipboardManager.current
                clipboard.setText(address)
```